src/main/resources/org/cbioportal/persistence/mybatisclickhouse/StudyViewMapper.xml

```diff
@@ -104,6 +104,21 @@
             <property name="table_name_prefix" value="sample"/>
         </include>
     </select>
+
+    <!-- for /sample-lists-counts/fetch (returns CaseListDataCount) -->
+    <select id="getCaseListDataCounts" resultType="org.cbioportal.model.CaseListDataCount">
+        SELECT
+            name AS label,
+            REPLACE(stable_id, CONCAT(cancer_study_identifier, '_'), '') AS value,
+            count(sample_id) AS count
+        FROM sample_list_list sll
+             LEFT JOIN sample_view sv ON sll.sample_id=sv.internal_id
+             LEFT JOIN sample_list sl on sll.list_id=sl.list_id
```

Comment:
I believe we have a sample_list_view or sample_list_columnstore that has sample to sample_list

Reply:
Curious to see timing of this query to using the one table sample_list_columnstore